flake.nix

```
@@ -32,7 +32,7 @@
 
         project = pkgs.haskell-nix.cabalProject' {
           src = ./.;
-          compiler-nix-name = "ghc926";
+          compiler-nix-name = "ghc947";
```

```suggestion
          compiler-nix-name = "ghc94";
```
just gets the latest 9.4 series